### ISSUE TITLE
Aperture tag stage style

### ISSUE THREAD

**Issue by FrozenDragon0** [BEEmod/BEE2-items#1328]
This would look like the stage maps near the end of aperture tag.

**Comment by TeamSpen210** [comment]
Definitely, but it would take a bit of work. All goo pits would convert. The outermost walls would be retextured to the 'stage' ones.

**Comment by Flametusk** [comment]
Except for the portalable ones, right? Because this would also be for regular bee2? And wow, the sunset on the ffxv title screen looks really cool.

**Comment by TeamSpen210** [comment]
Yes, portalable surfaces would be regular.

**Comment by Byzarru** [comment]
Some quick notes:
* The style should be called "Stage"
* Elevator room should be based on the underground vacuum terminal as seen in-game
* Broken parts of the wall and ceiling need to be manually placed
* The small obs. room should appear as the point lights as seen in the Stage chapter
* Other lights appear as normal
* A drop-down menu should appear in the Style pane to select which environment is going to be used. 
* TeamSpen210 mentioned in #902 that he needs to develop a 3D skybox for walls and ceilings to pull off the stage effect. How will this work when the walls are 2D images? (I don't know how these things work)
* A new wire item should be added to 'hang' items from ceilings, like catwalks, ramps (#1316) and diversity vents. This could be paired with my intangible scaffold idea (#1085). Two items would be laid down (one on the floor and one on the ceiling) and connected to each other.

**Comment by ZombiePwner1** [comment]
And there should be option allowing you to choose which stage you want such as snowy desert forest etc.

**Comment by Byzarru** [comment]
> And there should be option allowing you to choose which stage you want such as snowy desert forest etc.

I already mentioned that.
> A drop-down menu should appear in the Style pane to select which environment is going to be used.

**Comment by Luke18033** [comment]
Would this be available in regular Portal 2? Or would it just convert to Clean or something?

**Comment by Byzarru** [comment]
This should be available in Portal 2. A note I'd like to make is that all Stage maps are the same size and the test appears in the middle of the room. A wall skybox item would probably be the best means to achieve this effect.

**Comment by TeamSpen210** [comment]
Nope, what I'd do is take the outermost surfaces you make, and apply the stage textures to them. You just need to actually make the square room around your chamber - otherwise it would appear wrong.

**Comment by Endermage77** [comment]
Do you think we could apply the "Chamber inside the skybox" idea that Byzarru mentioned to the Old Aperture Enrichment Spheres?

**Comment by TeamSpen210** [comment]
The problem with enrichment spheres is actually that they're too small - a max-size chamber is bigger than a sphere.

**Comment by Flametusk** [comment]
Would it be possible to increase the size of the model?

**Comment by ZombiePwner1** [comment]
@Flametusk the problem with increasing the size is that the bigger they are the the bigger the shaft has to be and i'm not entirely sure if puzzlemaker maps can handle bigger underground shafts.

**Comment by Endermage77** [comment]
We could make the test shaft very foggy so we don't have to build as much of it. And we can cut out all the bits below the enrichment sphere you're in because I don't think you can see down there at any point.

**Comment by Byzarru** [comment]
I had once thought of an item that draws the Enrichment Sphere around the test chamber, allowing you to build your test within the size and space constraints of the sphere. This doesn't seem like something suited for BEE, though, and more along the lines of something @Luke18033 might develop for his PuzzleMaker Plus. 

The test shaft only appears in 80s Old Aperture, and only in a particular stylevar (unless I'm mistaken). 

Also, this thread is about Aperture Tag's stage style, which strictly puts itself in a square room and not the complex environment of Old Aperture.
